examples/distributed/kubernetes/k8s-manifests.yaml

```diff
@@ -0,0 +1,83 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: postgres-secrets
+type: Opaque
+stringData:
+  POSTGRES_PASSWORD: "superSecretPassword"
+  POSTGRES_USER: "optuna"
+  POSTGRES_DB: "optunaDatabase"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: postgres
+spec:
+  type: ClusterIP
+  selector:
+    app: postgres
+  ports:
+    - port: 5432
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: postgres
+  labels:
+    app: postgres
+spec:
+  selector:
+    matchLabels:
+      app: postgres
+  serviceName: postgres
+  template:
+    metadata:
+      labels:
+        app: postgres
+    spec:
+      containers:
+        - name: postgres
+          image: postgres:latest
+          imagePullPolicy: IfNotPresent
+          envFrom:
+            - secretRef:
+                name: postgres-secrets
+          ports:
+            - containerPort: 5432
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: worker
+spec:
+  parallelism: 5
+  template:
+    spec:
+      restartPolicy: OnFailure
+      initContainers:
+        - name: wait-for-database
+          image: postgres:latest
+          imagePullPolicy: IfNotPresent
+          command:
```

Comment:
How about using the list-style format like following?

https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/bc8e412670e5f8dd94189e80a0908d08ade196cc/cloudsql/mysql_wordpress_deployment.yaml#L44-L49

Reply:
It was just a matter of personal preference. I hope it's still readable enough